l10n_es_aeat_mod190/models/account_move_line.py

```diff
@@ -45,7 +45,6 @@ def _compute_aeat_perception_keys(self):
             aeat_perception_subkey_id = False
             if (
                 line.move_id.is_invoice()
-                and not line.exclude_from_invoice_tab
```

Comment:
Why is this removed?

Reply:
Removed exclude_from_invoice_tab boolean fields from version 16

Reply:
Yeah, but you have to look for the equivalent thing (`display_type`).

Reply:
But those are selection fields

Reply:
Well, I haven't dug enough, but there should be an equivalent logic. The question here is to not select more lines than the needed ones (and the same for the module part).